Directory.Build.props

```diff
@@ -33,8 +34,11 @@
     <IntermediateOutputPath Condition="'$(PlatformName)' != 'AnyCPU'">$(BaseIntermediateOutputPath)$(PlatformName)\$(Configuration)\</IntermediateOutputPath>
 
     <PackageOutputPath>$(ArtifactsPackagesDir)/$(MSBuildProjectName)</PackageOutputPath>
+
+    <!-- Used to help catch projects that use this property but don't end up importing our props below -->
+    <RequiredTargetFrameworks>non-existent-tfm</RequiredTargetFrameworks>
```

Comment:
Might be better for diagnostics to make it completely unusable but helpful, e.g. import-required-properties.